src/features/common/Layout/CustomTooltip/index.tsx

```diff
@@ -1,60 +1,49 @@
-import React from 'react';
+import { ReactElement, ReactNode } from 'react';
 import { bindHover, bindPopover } from 'material-ui-popup-state';
 import HoverPopover from 'material-ui-popup-state/HoverPopover';
 import { usePopupState } from 'material-ui-popup-state/hooks';
-import { IconProps } from '../../types/common';
 
 interface Props {
-  height: number;
-  width: number;
-  color: string;
-  children: React.ReactNode;
+  badgeContent: ReactElement;
+  children: ReactNode;
+  shouldDisplayPopup: boolean | undefined;
 }
```

Comment:
`badgeContent` could be renamed to `triggerElement` or `hoverTarget` to better reflect its role as the element that triggers the popup on hover.